README.md

```diff
@@ -20,16 +22,23 @@ A capstone project for UChicago's CAPP122 where abortion related data is taken f
 - `make visualize`: Takes the data produced from the `parse-data` layer and creates the project's visualizations
 
 ## How do we run this thing?
-For the `make visualize` command, you must have called the two commands that are referenced before it for it to run successfully.
-- Without the data from `make api` and `make parse-data`, `make visualize` has nothing to act on.
+There are two ways that you can run this application, one of them is to run all components of it at once and the other is to run each component individually. I will give you the instructions for both methods below.
 
-1. After you have installed [Poetry](https://python-poetry.org/docs/basic-usage/), run the command: `poetry shell`
+#### Run with one command
+1. After you have installed [Poetry](https://python-poetry.org/docs/basic-usage/), run the command from the base repository directory: `poetry shell`
+2. Run the command `poetry install` to install the package dependencies within the project.
+3. Run the `make run` command, and you will run the `api`, `data-parsing`, and `visualization` layers in serial. This will spin up a web page you can access via the URL `localhost:8005`.
```

Comment:
Does `make run` run everything at once without having to run anything manually (`make api`, etc.)? If so, it might be clearer to make this the wording: Run the `make run` command to run the `api`, `data-parsing`, and `visualization' layers in serial... (add additional information).

Also in terms of sequence of information, it makes more sense to have the run each package individually instructions to come before the run with one command instructions readers get an introduction to what the `api` etc. layers actually are/do before being provided with the option of the run all command.

Reply:
I took out the initial mentions of the other layers. Given that, I think it makes sense to have the general case first (`make run`) and the more advanced nuanced case second (individual parts).